.github/workflows/Tests.yaml

```diff
@@ -534,7 +526,7 @@ jobs:
           -D CHARM_ROOT=/work/charm_6_10_2/multicore-linux-x86_64-${CHARM_CC}
           -D CMAKE_BUILD_TYPE=${{ matrix.build_type }}
           -D DEBUG_SYMBOLS=OFF
-          -D RUN_TESTS_IN_TEST_EXECUTABLES=OFF
+          -D UNIT_TESTS_IN_TEST_EXECUTABLES=OFF
```

Comment:
Might as well delete this line, since it doesn't do anything here. Up to you 🤷‍♂️

Reply:
I personally would find it less confusing if the two CMake calls are the same. E.g. if I look at this again in a year or two from now I don't have to think about why the calls are different in this regard, but also :shrug: